packages/frontend/src/components/atoms/spinner.atom.tsx

```
@@ -0,0 +1,28 @@
+export function Spinner({ size = 'md', className }: SpinnerProps) {
+  const sizeClasses = {
+    xs: 'h-3 w-3',
+    sm: 'h-4 w-4',
+    md: 'h-6 w-6',
+    lg: 'h-8 w-8',
+    xl: 'h-12 w-12',
+  };
+
+  return (
+    <svg className={cn('text-primary-600 dark:text-primary-500 animate-spin', sizeClasses[size], className)} xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
+      <circle className="opacity-25" cx="12" cy="12" r="10" stroke="currentColor" strokeWidth="4" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)